draft-ietf-rtcweb-jsep.xml

```diff
@@ -448,11 +448,15 @@ setRemote(ANSWER) |                                   |
           property is non-null; otherwise it is said to be
           disassociated. The associated m= section is determined using
           a mapping between transceivers and m= section indices, formed
-          when creating an offer or applying a remote offer. An
-          RtpTransceiver is never associated with more than one m=
+          when creating an offer or applying a remote offer.</t>
+
+          <t>An RtpTransceiver is never associated with more than one m=
           section, and once a session description is applied, a m=
           section is always associated with exactly one
-          RtpTransceiver.</t>
+          RtpTransceiver. However, once a RtpTransceiver is stopped,
+          and/or its associated m= section is rejected, that m= section
+          may be "recycled" and associated with a new RtpTransceiver with
```

Comment:
This "may" is making me a little sad, because we are very strict about what you can and cannot do.

Perhaps "will, under certain conditions" and then a cite to the relevant sections?